<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalFunctions</class>
 <widget class="QDialog" name="AdditionalFunctions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AdditionalFunctions {
background-color:  rgb(139, 142, 161);
}</string>
  </property>
  <widget class="QSpinBox" name="spinBox_koriza">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 10px;
background: #41bfbe;
margin: 0 -4px; 
font-size: 35px;
color: rgb(51, 40, 48);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_coffee">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 10px;
background: #41bfbe;
margin: 0 -4px; /* расширяется наружу от бороздки */
font-size: 35px;
color: rgb(51, 40, 48);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_milk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 10px;
background: #41bfbe;
margin: 0 -4px; 
font-size: 35px;
color: rgb(51, 40, 48);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_milk">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Button_milk{
transition-duration: 0.8s;
border-radius: 0px;
background-color: rgb(80, 78, 99);
font-size: 20px;
color:rgb(254, 251, 236);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;

}

#Button_milk:hover {
background-color:rgb(42, 41, 53);
}</string>
   </property>
   <property name="text">
    <string>ДОДАТИ</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_water">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 10px;
background: #41bfbe;
margin: 0 -4px; /* расширяется наружу от бороздки */
font-size: 35px;
color: rgb(51, 40, 48);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_water">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Button_water{
transition-duration: 0.8s;
border-radius: 0px;
background-color: rgb(80, 78, 99);
font-size: 20px;
color:rgb(254, 251, 236);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}

#Button_water:hover {
background-color:rgb(42, 41, 53);
}</string>
   </property>
   <property name="text">
    <string>ДОДАТИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;г&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_on_mainwindow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>791</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Button_on_mainwindow {
transition-duration: 0.8s;
border-radius: 10px;
background-color: rgb(80, 78, 99);
font-size: 20px;
color:rgb(254, 251, 236);
	font: 75 20pt &quot;MS Shell Dlg 2&quot;;
}

#Button_on_mainwindow:hover {
background-color:rgb(42, 41, 53);
}</string>
   </property>
   <property name="text">
    <string>Повернутися назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;мл&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;г&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-color: rgb(51, 40, 48);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;Додати&lt;/span&gt;&lt;/h2&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt; молоко&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_coffee">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Button_coffee{
transition-duration: 0.8s;
border-radius: 0px;
background-color: rgb(80, 78, 99);
font-size: 20px;
color:rgb(254, 251, 236);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}

#Button_coffee:hover {
background-color:rgb(42, 41, 53);
}</string>
   </property>
   <property name="text">
    <string>ДОДАТИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-color: rgb(51, 40, 48);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;Додати&lt;/span&gt;&lt;/h2&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt; каву&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>270</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;г&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_koriza">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Button_koriza{
transition-duration: 0.8s;
border-radius: 0px;
background-color: rgb(80, 78, 99);
font-size: 20px;
color:rgb(254, 251, 236);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}

#Button_koriza:hover {
background-color:rgb(42, 41, 53);
}</string>
   </property>
   <property name="text">
    <string>ДОДАТИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_sugar">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Button_sugar{
transition-duration: 0.8s;
border-radius: 0px;
background-color: rgb(80, 78, 99);
font-size: 20px;
color:rgb(254, 251, 236);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}

#Button_sugar:hover {
background-color:rgb(42, 41, 53);
}</string>
   </property>
   <property name="text">
    <string>ДОДАТИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-color: rgb(51, 40, 48);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;Додати&lt;/span&gt;&lt;/h2&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;корицю&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sugar">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">height: 10px;
background: #41bfbe;
margin: 0 -4px; 
font-size: 35px;
color: rgb(51, 40, 48);</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-color: rgb(51, 40, 48);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;Додати&lt;/span&gt;&lt;/h2&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt; воду&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>791</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(51, 40, 48);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;Додайте необхідні для приготування продукти&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;мл&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px 0 0 0;
background-color: rgb(51, 40, 48);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;Додати&lt;/span&gt;&lt;/h2&gt;&lt;h2 align=&quot;center&quot; style=&quot; margin-top:16px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#fefbec;&quot;&gt;цукор&lt;/span&gt;&lt;/h2&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>21</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;г&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
